src/API/useInAppUpload.js

```diff
@@ -2,17 +2,26 @@ import { useMutation } from 'react-query';
 
 import { useOkapiKy } from '@folio/stripes/core';
 
+import { CAPABILITIES, CAPABILITIES_VALUE } from '../constants';
+
 export const useInAppUpload = () => {
   const ky = useOkapiKy();
 
-  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates }) => {
-    return ky.post(`bulk-edit/${jobId}/item-content-update/upload`, {
-      searchParams: { limit: 10 },
-      json: {
-        itemContentUpdates: contentUpdates,
-        totalRecords: contentUpdates.length,
-      },
-    }).json();
+  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates, capability }) => {
+      const typeOfBulk = capability === CAPABILITIES.ITEM ?
```

Comment:
I'd suggest to have a map between capabilities and capability values instead of conditions

Reply:
@NikitaSedyx , can you explain what do you mean when talking about  a map between capabilities and capability values ?